assets/init

```diff
@@ -369,6 +370,12 @@ if [ "${SMTP_ENABLED}" == "true" ]; then
     "") sudo -u redmine -H sed '/{{SMTP_AUTHENTICATION}}/d' -i config/initializers/smtp_settings.rb ;;
     *) sudo -u redmine -H sed 's/{{SMTP_AUTHENTICATION}}/'"${SMTP_AUTHENTICATION}"'/' -i config/initializers/smtp_settings.rb ;;
   esac
+
+  case "${SMTP_METHOD}" in
```

Comment:
@fbertux this else condition is not required. simply add the `
*) sudo -u redmine -H sed 's/{{SMTP_AUTHENTICATION}}/'"${SMTP_AUTHENTICATION}"'/g' -i config/initializers/smtp_settings.rb` line at https://github.com/sameersbn/docker-redmine/blob/master/assets/init#L346

Notice that i have add the `g` option at the end of your `sed` command. It is required when trying to change more than matches in `sed` which in your case are two instances of the string `{{SMTP_METHOD}}`.

Please update the PR and let me know. Please make sure you test it out before updating the PR. Thanks.